site/design-notes/patterns/towards-member-patterns.md

```diff
@@ -492,4 +492,4 @@ objects.match(String::instanceof)
 
 where `String::instanceof` means the same as the previous lambda example.  This
 means that APIs like `Stream` can abstract over conditional behavior as well as
-unconditional.
\ No newline at end of file
+unconditional.
```

Comment:
Whoops, that was unintentional. Lmk if you all want me to fix this, but since it is Markdown, it should not affect the output at all.